pkgs/tools/misc/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchFromGitHub, rustPlatform, cmake, perl, pkgconfig, zlib
+, darwin, libiconv
+}:
+
+with rustPlatform;
+
+buildRustPackage rec {
+  name = "sd-${version}";
+  version = "0.4.3";
+
+  cargoSha256 = "08zzn3a32xfjkmpawcjppn1mr26ws3iv40cckiz8ldz4qc8y9gdh";
```

Comment:
This hash is from the `exa` package. Should be changed to the correct value that you can get from a failed build, like [described in the manual](https://nixos.org/nixpkgs/manual/#compiling-rust-applications-with-cargo).